File: app/src/main/java/com/shumidub/todoapprealm/realmmodel/task/FolderTaskObject.java
package com.shumidub.todoapprealm.realmmodel.task;

import io.realm.RealmList;
import io.realm.RealmObject;

/**
 * Created by Артем on 24.12.2017.
 */

public class FolderTaskObject extends RealmObject {

    private String name;
    private long id;
    public io.realm.RealmList<TaskObject> folderTasks;
    public boolean isDaily;

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }

    public long getId() {
        return id;
    }
    public void setId(long id) {
        this.id = id;
    }

    public RealmList<TaskObject> getTasks() {
        return folderTasks;
    }
    public void setTasks(RealmList<TaskObject> tasks) {
        this.folderTasks = folderTasks;
    }

    public boolean isDaily() {
        return isDaily;
    }

    public void setDaily(boolean daily) {
        isDaily = daily;
    }
}

File: app/src/main/java/com/shumidub/todoapprealm/realmmodel/task/FolderTaskObject.kt
package com.shumidub.todoapprealm.realmmodel.task

import io.realm.kotlin.ext.realmListOf
import io.realm.kotlin.types.RealmList
import io.realm.kotlin.types.RealmObject

class FolderTaskObject : RealmObject {
    var id: Long = 0
    var name: String? = null
    var folderTasks: RealmList<TaskObject> = realmListOf()
    var isDaily: Boolean = false
}
